go.mod

```diff
@@ -338,7 +339,7 @@ require (
 	github.com/AzureAD/microsoft-authentication-library-for-go v0.7.0 // indirect
 	github.com/Masterminds/sprig/v3 v3.2.2
 	github.com/Microsoft/go-winio v0.6.0 // indirect
-	github.com/ProtonMail/go-crypto v0.0.0-20210428141323-04723f9f07d7
+	github.com/ProtonMail/go-crypto v0.0.0-20230426101702-58e86b294756
```

Comment:
with this update, I was able to properly use errors returned by the `opengpg` library and compare them with  `errors.Is(err, ErrRevokedKey)`